frontend/demo/src/model/Tag.ts

```diff
@@ -0,0 +1,46 @@
+export interface Tag {
+  id?: string;
+  name: string;
+  color: string;
+  count?: number;
+}
+
+export interface TagPayload {
+  id: string;
+}
+
+export interface CreateTagRequestPayload {
+  name: string;
+  color: string;
+}
+
+export interface GetTagsResponse {
+  data: Tag[];
+}
+
+export const tagsMapper = (serverTags: Tag[]): Tag[] => {
+  const tags: Tag[] = [];
+  const tags2 = serverTags.map((tag: Tag) => {
+    tag.color = colorMapper(tag.color);
+    tags.push(tag);
+  });
+  return tags;
+};
+
+export const colorMapper = (color: string): string => {
+  switch (color) {
```

Comment:
maybe add a `default` in case the API returns something unexpected. With a default won't break the app